app/[lang]/error.tsx

```diff
@@ -40,7 +42,7 @@ export default function Error({
         </h2>
 
         <div className=" mt-8 w-full text-center font-cbrush text-2xl md:w-auto">
-          <Button href="/">{t('error.back_safety')}</Button>
+          <Button onClick={() => router.reload()}>{t('error.reload')}</Button>
```

Comment:
As well, the translation data 'error.reload` would not return any valid copy as the key needs to be changed to match in `/i18n/locales/en.ts`

Reply:
Thank you for the heads up. Fixed this.